benchmarks/src/jmh/java/org/corfudb/benchmarks/cluster/ClusterBenchmark.java

```diff
@@ -0,0 +1,212 @@
+package org.corfudb.benchmarks.cluster;
+
+import static org.corfudb.benchmarks.util.DataUnit.KB;
+import static org.corfudb.benchmarks.util.DataUnit.MB;
+import static org.corfudb.universe.scenario.fixture.Fixtures.TestFixtureConst.DEFAULT_STREAM_NAME;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.corfudb.benchmarks.util.DataGenerator;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.universe.UniverseManager;
+import org.corfudb.universe.UniverseManager.UniverseWorkflow;
+import org.corfudb.universe.group.cluster.CorfuCluster;
+import org.corfudb.universe.node.client.CorfuClient;
+import org.corfudb.universe.universe.Universe.UniverseMode;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+import org.openjdk.jmh.results.format.ResultFormatType;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.TimeValue;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+
+@Slf4j
+public class ClusterBenchmark {
```

Comment:
Can you please explain what the test does? What is its purpose?

Reply:
done